scripts/core/editor3/components/Editor3.jsx

```diff
@@ -173,9 +174,25 @@ export class Editor3Component extends React.Component {
         case 'soft-newline':
             newState = RichUtils.insertSoftNewline(editorState);
             break;
+        case 'delete':
+            if (!this.allowEditSuggestion('delete')) {
```

Comment:
I wish it were possible to use an abstraction for deletion like `handleBeforeInput`. Since we aren't using one, we are limited to supporting only two ways of removing text. For example when using `ctrl+x` the text is removed, but not marked as a deletion.

Reply:
Fixed ctrl-x issue.